xword_dl.py

```diff
@@ -28,7 +28,7 @@ def save_puzzle(puzzle, filename):
         print("Not saving: a file named {} already exists.".format(filename))
 
 def remove_invalid_chars_from_filename(filename):
-    invalid_chars = '<>:"/\|?*'
+    invalid_chars = r'<>:"/\|?*'
```

Comment:
My IDE, PyCharm, showed a tooltip warning here: 

> PEP 8: W605 invalid escape sequence '\|'

Adding the `r` turns this into a "raw string" so it doesn't treat the backslash as an escape.